### PR TITLE
Allow to configure ES index suffix

### PR DESCRIPTION
**Pull request by ChrisKujawa**
## Description

Previously we always wrote the `yyyy-MM-dd` as suffix to every index. This was hard-coded and not possible to configure/overwrite, which can be seen as a limitation. This makes it hard to delete indices via ILM, especially on higherload. We have run into this problem with our benchmarks.

This PR makes it possible to configure the default suffix format 'yyyy-MM-dd' and overwrite it with something like: 'yyyy-MM-dd_HH' which allows the creation of indices based on hours.

But it is also possible to create indices per day, if necessary when not much data is expected.

This configuration makes it possible to better leverage the ILM settings provided by ES.

Yesterday, I ran it to see it in action.

es-indices

## Related issues

relates to https://github.com/zeebe-io/benchmark-helm/issues/127
relates to https://github.com/zeebe-io/benchmark-helm/issues/126

### CODE REVIEW

**Review by npepinpe — APPROVED** [review]
:+1:

can you check if we need to update the docs page? would be good to reference the pattern iso number and/or the reference elastic docs page

if there is no camunda docs page for it, lets at least document it in our template files

File: dist/src/main/config/broker.standalone.yaml.template [hunk 1]
```diff
@@ -749,6 +749,7 @@
         #   index:
         #     prefix: zeebe-record
         #     createTemplate: true
+        #     indexSuffixDatePattern = "yyyy-MM-dd"
```
Comment (Zelldon) [inline dist/src/main/config/broker.standalone.yaml.template:752]: @npepinpe I did here or what do you mean?

Comment (Zelldon) [reply dist/src/main/config/broker.standalone.yaml.template:752]: We could like to https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html ?

**Comment by Zelldon** [comment]
I will update https://docs.camunda.io/docs/self-managed/zeebe-deployment/exporters/elasticsearch-exporter/ as well then

File: dist/src/main/config/broker.standalone.yaml.template (hunk 1 above)
Comment (Zelldon) [reply dist/src/main/config/broker.standalone.yaml.template:752]: Since the formatter seems to support several iso formats

**Comment by Zelldon** [comment]
https://github.com/camunda/camunda-docs/pull/3216

**Pull request merged**